package.json

```diff
@@ -38,7 +38,9 @@
     "flow-bin": "^0.79.1",
     "husky": "^0.14.3",
     "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "react-dom": "^16.4.2",
+    "scrollreveal": "^4.0.0",
```

Comment:
Are we adding this dependency just to be used in the stories?
Is it included in the dist?

Reply:
It's a dev dependency. Actually, we have no dependencies in this project :)